### PR TITLE
Tidy up additional features

### PR DESCRIPTION
**Pull request by danieldortega77**
The features added are:
- Comment section on all pages
- Loading message before comment section appears on page

Code was refactored so that the navbar, social media icons, and comment section are in separate HTML files. These HTML files are injected into the website's pages with JS.

### CODE REVIEW

File: portfolio/src/main/webapp/comments.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  </div>
+  <div class="row" id="comment-section">
+    <div class="col-md-6">
+      <label for="max-comments">Number of comments:</label>
+      <input type="number" id="max-comments" name="max-comments" min="1" max="100" value="5" onchange="getComments()"/>
+      <div id="comments-list"></div>
+      <input type="submit" id="delete-button" value="Delete all comments" onclick="deleteComments()"/>
+    </div>
+  </div>
+</div>
```
Comment (dslucas) [inline portfolio/src/main/webapp/comments.html]: nit: Let's add a newline at the end of this file so the red symbol goes away.

File: portfolio/src/main/webapp/socials.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        <img src="/images/Instagram.png">
+      </a>
+    </div>
+    <div class="col-4 col-md-2">
+      <a href="https://www.facebook.com/profile.php?id=100025246023927">
+        <img src="/images/Facebook.png">
+      </a>
+    </div>
+  </div>
+</div>
```
Comment (dslucas) [inline portfolio/src/main/webapp/socials.html]: Same here.

File: portfolio/src/main/webapp/navbar.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" href="work.html">Experience</a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link" href="images.html">Images</a>
+    </li>
+    </ul>
+  </div>
+</nav>
```
Comment (dslucas) [inline portfolio/src/main/webapp/navbar.html]: Same here.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -12,6 +12,30 @@
...
+/**
+ * Loads all elements of the page that are factored out originally.
+ */
+async function loadElements(page) {
+  await htmlInject('navbar.html', 'navbar-container');
+  var navbarOptions = document.querySelectorAll('.nav-item')
+  navbarOptions[page].classList.add("active");
+  await htmlInject('socials.html', 'socials-container');
+  await htmlInject('comments.html', 'comments-container');
+  getComments();
```
Comment (dslucas) [inline portfolio/src/main/webapp/script.js]: I think this function and code could benefit from some commenting (e.g. what the `page` parameter is and what it's for, why we're calling `htmlInject` twice, etc.).

**Review by dslucas — COMMENTED** [review]
Cool stuff! As you probably experienced, making a website/webapp dynamic can get a bit tedious. That's actually a primary motivation why front end developers often use frameworks like Angular or React or libraries like jQuery: they abstract away a lot of that manual JavaScript/HTML work.

File: portfolio/src/main/webapp/comments.html (hunk 1 above)
Comment (danieldortega77) [reply portfolio/src/main/webapp/comments.html]: Done.

File: portfolio/src/main/webapp/socials.html (hunk 2 above)
Comment (danieldortega77) [reply portfolio/src/main/webapp/socials.html]: Done again

File: portfolio/src/main/webapp/navbar.html (hunk 3 above)
Comment (danieldortega77) [reply portfolio/src/main/webapp/navbar.html]: Done once again

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (danieldortega77) [reply portfolio/src/main/webapp/script.js]: Good idea, as the `page` parameter's function is unclear here.

**Review by dslucas — APPROVED** [review]

**Review by dborkan — APPROVED** [review]
👍 nice use of templates

**Pull request merged**